src/dataaccess/GistAdjuvantIhanos.json

```diff
@@ -1132,6 +1146,13 @@
                     }
                 }
             }
+        },
+        "SourceClinicalNote": {
+            "_ShrId": "788dcbc3-ed18-470c-89ef-35ff91854c7f",
+            "_EntryId": "5160",
```

Comment:
This is actually the note that is requesting the surgery so it should be the source node for the Procedure Performed for the surgery not the CT-guided biopsy